src/shapes/polyline.class.ts

```diff
@@ -170,27 +205,42 @@ export class Polyline extends FabricObject {
    */
   _getNonTransformedDimensions() {
     return this.exactBoundingBox
-      ? new Point(this.width, this.height)
+      ? // TODO: fix this
+        new Point(this.width, this.height)
       : super._getNonTransformedDimensions();
   }
 
   /**
    * @override stroke and skewing are taken into account when projecting stroke on points,
-   * therefore we don't want the default calculation to account for skewing as well
+   * therefore we don't want the default calculation to account for skewing as well.
+   * Though it is possible to pass `width` and `height` in `options`, doing so is very strange, use with discretion.
```

Comment:
Once your suggestion regarding width/height will be fulfilled this line should be removed